lita-jarvis.gemspec

```diff
@@ -25,10 +25,11 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "mbox"
   spec.add_runtime_dependency "cabin"
   spec.add_runtime_dependency "open4"
-  #spec.add_runtime_dependency "lita-hipchat"
+  spec.add_runtime_dependency "lita-hipchat"
```

Comment:
Adding this causes Lita to somehow explicitly try to use hipchat, so I can't test with the shell adapter anymore:

```
% bundle exec lita
F, [2016-01-07T15:18:07.008821 #7190] FATAL -- lita: Configuration attribute "jid" is required for "hipchat" adapter.
```

^^ the above on my local system used to run Lita under the Shell adapter, but it now fails. Related to https://github.com/litaio/lita-hipchat/issues/33 which I'm hoping to make time to solve soon.